### PR TITLE
Lab5: Implement stringer solution and tests

### LINKED ISSUE #528: Lab5 - Stringer (alanhillgemann)

### PR DESCRIPTION
**Pull request by alanhillgemann**
Fixes #528 

Review of colleague's PR #501 

#### Changes proposed in this PR:
- Implement stringer solution and tests

### CODE REVIEW

File: 05_stringer/alanhillgemann/main.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+type IPAddr [4]byte
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, IPAddr{127, 0, 0, 1})
+
+}
+
+func (ip IPAddr) String() string {
```
Comment (anzdaddy) [inline 05_stringer/alanhillgemann/main.go]: Keep methods close to their type.

**Review by anzdaddy — DISMISSED** [review]

File: 05_stringer/alanhillgemann/main.go (hunk 1 above)
Comment (alanhillgemann) [reply 05_stringer/alanhillgemann/main.go]: Fixed

**Comment by alanhillgemann** [comment]
@anzdaddy thank you for reviewing my PR. I have made all changes requested.

**Review by juliaogris — DISMISSED** [review]
I've updated the Markdown formatting on your PR description and you still need to add a link to your review of a colleague's PR please.

Other than that: LGTM

**Comment by alanhillgemann** [comment]
@juliaogris thank you for your review. I have fixed your feedback item. Could you please review again?

**Comment by harirakr** [comment]
@alanhillgemann Make sure you change the label on the PR to "Ready for Review" when you request re-review. 
Please check merge and tests after clicking "Update Branch".

**Review by rokane — APPROVED** [review]
Looks good to me! :)

**Pull request merged**